x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -619,5 +619,86 @@ export function MachineLearningJobWizardCommonProvider(
         await testSubjects.existOrFail(expectedSelector);
       });
     },
+
+    async togglingModelChangeAnnotationsShowsCalloutAndRemovesCallout() {
+      await mlCommonUI.toggleSwitchIfNeeded('mlJobWizardSwitchAnnotations', false);
+      await testSubjects.existOrFail('mlJobWizardAlsoEnableAnnotationsRecommendationCallout', {
+        timeout: 3_000,
+      });
+      await mlCommonUI.toggleSwitchIfNeeded('mlJobWizardSwitchAnnotations', true);
+      await testSubjects.missingOrFail('mlJobWizardAlsoEnableAnnotationsRecommendationCallout', {
+        timeout: 3_000,
+      });
+    },
+
+    async goToTimeRangeStep() {
+      await testSubjects.existOrFail('mlJobWizardTimeRangeStep', {
+        timeout: 3_000,
+      });
+      await testSubjects.click('mlJobWizardTimeRangeStep');
+      await testSubjects.existOrFail('mlJobWizardStepTitleTimeRange', {
+        timeout: 3_000,
+      });
+    },
+
+    async assertShortDurationTimeRange() {
+      const { startDate: origStartDate } = await this.getSelectedDateRange();
+
+      // calculate the new end datedate
+      const shortDurationEndDate = `${origStartDate?.split(':', 1)[0]}:01:00.000`;
+
+      // set the new end date
+      await testSubjects.setValue('mlJobWizardDatePickerRangeEndDate', shortDurationEndDate, {
+        clearWithKeyboard: true,
+        typeCharByChar: true,
+      });
+
+      // click away from time popover
+      await this.goToTimeRangeStep();
+
+      // assert time is set as expected
+      await this.assertDateRangeSelection(origStartDate as string, shortDurationEndDate);
+    },
```

Comment:
The entire method is misleading. You are not supposed to set any values in assertion methods. You can consider setting a values in the test suite itself and then asset the callout is visible.

Reply:
https://github.com/elastic/kibana/commit/dfa29d5957e95836cef92d53ade61c9fb5e06254

Reply:
https://github.com/elastic/kibana/commit/dfa29d5957e95836cef92d53ade61c9fb5e06254